server/database/queries/products/addNewProductQuery.js

```diff
@@ -0,0 +1,9 @@
+const connection = require('../../config/connection');
+
+const addNewProductQuery = (
+  name, price, discount, url, category,
+) => connection.query(
+  'INSERT INTO products (name, price, discount, image, category_id) VALUES ($1, $2, $3, $4, $5) RETURNING *',
```

Comment:
Why would you return the inserted data without any reason?
you are not going to use this data so you don't need to return anything
```suggestion
  'INSERT INTO products (name, price, discount, image, category_id) VALUES ($1, $2, $3, $4, $5);',
```